library/src/main/java/com/pokegoapi/api/gym/Battle.java

```diff
@@ -108,10 +102,10 @@ public Result start() throws LoginFailedException, RemoteServerException {
 			gymIndex.add(action.getTargetIndex());
 		}
 
-		return battleResponse.getResult();
+		return battleResponse.getResult();*/
+		return null;
```

Comment:
as said on previous post, i was not able to kaintain battle code. it was never working on my side, but feel free to contrib if you want